resources/prometheus/prometheus-rules.yaml

```diff
@@ -176,7 +176,68 @@ spec:
             summary: "Fleetshard synchronizer manages `{{ $value }}` centrals."
             description: "Fleetshard synchronizer manages `{{ $value }}` centrals. The number of Centrals should always be larger than zero in a working system. If it drops to or below zero, fleetshard synchronizer is assumed to be in a failed state."
             sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-007-fleetshard-sync-reconciliation-error.md"
-
+    - name: rhacs-aws-quota
+      rules:
+        - alert: RHACSCentralDBClustersUtilizationHigh
+          expr: |
+            acs_fleetshard_central_db_clusters_used / acs_fleetshard_central_db_clusters_max >= 0.8
+          for: 1h
+          labels:
+            severity: warning
+          annotations:
+            summary: "The number of RDS DB clusters is close to its limit."
+            description: "The number of AWS RDS DB clusters is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBInstancesUtilizationHigh
+          expr: |
+            acs_fleetshard_central_db_instances_used / acs_fleetshard_central_db_instances_max >= 0.8
+          for: 1h
+          labels:
+            severity: warning
+          annotations:
+            summary: "The number of RDS DB instances is close to its limit."
+            description: "The number of AWS RDS DB instances is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBManualSnapshotsUtilizationHigh
+          expr: |
+            acs_fleetshard_central_db_snapshots_used / acs_fleetshard_central_db_snapshots_max >= 0.8
+          for: 5m
+          labels:
+            severity: warning
+          annotations:
+            summary: "The number of RDS DB snapshots is close to its limit."
+            description: "The number of AWS RDS DB manual snapshots is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBClustersUtilizationCritical
+          expr: |
+            acs_fleetshard_central_db_clusters_used / acs_fleetshard_central_db_clusters_max >= 0.9
```

Comment:
Is 0.9 too low for a critical alert? I could change this to 0.95 for example.

Triggering it only when `acs_fleetshard_central_db_clusters_used = acs_fleetshard_central_db_clusters_max` would be too late IMO, as it takes a few hours (up to 4) for AWS to react to a quota increase request.

Reply:
What is the limit? In other words, how many remaining instances do we have left at 90%?

Reply:
It depends on the current maximum. Which maximum can be increased when the actual usage is close to it, otherwise AWS refuse to increase it. Hence this alert.

Do you think we should rather have this on concrete values than percentages?

E.g. `acs_fleetshard_central_db_clusters_max - acs_fleetshard_central_db_clusters_used < 10`?

Reply:
I think it would be good now how many slots are still left. But we could also add this info (`acs_fleetshard_central_db_clusters_max - acs_fleetshard_central_db_clusters_used`) to the alert description.

Reply:
Added `acs_fleetshard_central_db_clusters_max - acs_fleetshard_central_db_clusters_used` to the alert description.